src/prisma/schema.prisma

```diff
@@ -1323,6 +1323,7 @@ model ProposalEvaluation {
   permissions          ProposalEvaluationPermission[]
   proposalNotification ProposalNotification[]
   actionLabels         Json?
+  failReasons          String[]                            @default([])
```

Comment:
is this a list to future-proof it to allow multiple reasons? seems a little over-engineering but is ok

Reply:
maybe suggestion: rename this to "decisionReasons" if we are trying to future-proof, maybe they'll want reasons for success